libs/libssh/Makefile

```diff
@@ -7,24 +7,22 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_LICENSE:=LGPL-2.1-or-later BSD-2-Clause
-PKG_MAINTAINER:=Mislav Novakovic <mislav.novakovic@sartura.hr>
-
 PKG_NAME:=libssh
 PKG_VERSION:=0.9.3
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.libssh.org/files/0.9/
 PKG_HASH:=2c8b5f894dced58b3d629f16f3afa6562c20b4bdc894639163cf657833688f0c
 
+PKG_MAINTAINER:=Mislav Novakovic <mislav.novakovic@sartura.hr>
+PKG_LICENSE:=LGPL-2.1-or-later BSD-2-Clause
+PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:libssh:libssh
 
 CMAKE_INSTALL:=1
 CMAKE_BINARY_SUBDIR:=build
-
 PKG_BUILD_PARALLEL:=1
-PKG_USE_MIPS16:=0
```

Comment:
Shouldn't we test whether the outcome works with mips16? Is there some particular reason you need to use this instruction set?

Reply:
I think it should be the other way around

Most other packages pass this variable as they want MIPS32 for performance reasons. I don't know that libssh needs performance. The crypto is handled by either mbedTLS or OpenSSL.

Reply:
I feel we are bikeshedding as I see little value in this discussion. The flag is already there and the package is tested with it. I'm happy to add it if you provide some data that first it works, and that second there are not performance issues there.

Reply:
Removed the removal.